### PR TITLE
Fix wrong href.

### PR DESCRIPTION
**Pull request by afranjin**
The href path of don't delete button needs to be ../../../

#1000

### CODE REVIEW

File: helpdesk/templates/helpdesk/ticket_dependency_del.html [hunk 1]
```diff
@@ -18,7 +18,7 @@ <h2>Delete Ticket Dependency</h2>
 <p>Are you sure you wish to remove the dependency on this ticket?</p>
 {% endblocktrans %}
 
-<p><a href='../../'><button class="btn btn-primary btn-lg">{% trans "Don't Delete" %}</button></a></p>
+<p><a href='../../../'><button class="btn btn-primary btn-lg">{% trans "Don't Delete" %}</button></a></p>
```
Comment (Benbb96) [inline helpdesk/templates/helpdesk/ticket_dependency_del.html:21]: Maybe we should use `{{ ticket.get_absolute_url }}` instead of a a relative URL.

Comment (afranjin) [reply helpdesk/templates/helpdesk/ticket_dependency_del.html]: Changed in 3ea2d1dd501b249c43f1c2ed2ce67f09c6e615d6

File: helpdesk/templates/helpdesk/ticket_dependency_del.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
     <a href="{% url 'helpdesk:list' %}">{% trans "Tickets" %}</a>
 </li>
 <li class="breadcrumb-item">
-    <a href="{% url 'helpdesk:list' %}{{ ticket.id }}/">{{ ticket.queue.slug }}-{{ ticket.id }}</a>
+    <a href="{% url 'helpdesk:list' %}{{ dependency.ticket.id }}/">{{ dependency.ticket.queue.slug }}-{{ dependency.ticket.id }}</a>
```
Comment (Benbb96) [inline helpdesk/templates/helpdesk/ticket_dependency_del.html:10]: Good catch, I didn't see that ticket was not available in the template. You can replace `{% url 'helpdesk:list' %}{{ dependency.ticket.id }}/` by `{{ dependency.ticket.get_absolute_url }}` 😉

**Review by Benbb96 — APPROVED** [review]

**Comment by gwasser** [comment]
Thank you @afranjin and @Benbb96 for working on this! Makes it easy for me to look at and approve.

**Pull request merged**